### PR TITLE
Create a Giter8 template for new LaTiS server projects

### PR DESCRIPTION
**Pull request by lindholc**
Look in the `src/main/g8` directory to see the files that will be copied by giter8. (With the exception of `default.properties`, which is for giter8.)

The only things being filled by giter8 are the name of the directory the project is placed in, the name of the project in `build.sbt`, and the `version` key in `latis.properties`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# LaTiS Server Project Template
+
+A [Giter8][g8] template for creating runnable LaTiS servers.
```
Comment (lindholc) [inline README.md]: I don't know about the wording here. I also used similar wording in the description of this repository and the `description` in `default.properties`.

File: src/main/g8/build.sbt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+lazy val root = (project in file(".")).
+  dependsOn(latis).
+  settings(
+    name := "$name;format="normalized"$",
```
Comment (lindholc) [inline src/main/g8/build.sbt:4]: Here is one spot where we substitute the name of the project.

File: src/main/g8/src/main/resources/latis.properties [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# LaTiS properties
+
+version = $name;format="normalize"$-SNAPSHOT
```
Comment (lindholc) [inline src/main/g8/src/main/resources/latis.properties:3]: Here is one spot where we substitute the name of the project.

File: README.md (hunk 1 above)
Comment (dlindhol) [reply README.md]: How about "a" ... "server"?

Comment (dlindhol) [reply README.md]: Also move stuff about running the server into a README that gets generated.

**Pull request merged**